src/etc/inc/filter.inc

```diff
@@ -3397,6 +3397,7 @@ EOD;
 				}
 				/* allow access to DHCP relay on interfaces */
 				if (isset($config['dhcrelay']['enable'])) {
+					$dhcrelaycfg =& $config['dhcrelay'];
 					$dhcifaces = explode(",", $dhcrelaycfg['interface']);
```

Comment:
I don't see a reason to set a reference to $config['dhcrelay']. IMO this variable is unnecessary and $config['dhcrelay'] can be used direct on explode() call